apps/test/unit/applab/applabTest.js

```diff
@@ -522,6 +522,18 @@ describe('applab', () => {
     });
   });
 
+  describe('Turtle mode in debugger', () => {
+    it('will slow the speed of how fast each block is run in the workspace', () => {
+      Applab.setStepSpeed(0);
+      assert.equal(Applab.scale.stepSpeed, 1500);
+    });
+
+    it('will  each block is run in the workspace', () => {
```

Comment:
tiny nit:  on line 531, what are you checking for your second test case (i.e the it statement)

Reply:
Woops! I was going to add more to that but forgot! I'll make the change now.